authorization/src/lib.rs

```diff
@@ -49,7 +49,7 @@ impl NonceAuth for WrappedAuth {
 
 pub struct AuthContract;
 
-#[contractimpl(export_if = "export")]
+#[contractimpl(export)]
```

Comment:
Export is enabled by default, you should only need to specify it if you want to disable it.
```suggestion
#[contractimpl]
```